visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1975,6 +1975,65 @@ namespace pcl
       protected:
         /** \brief The render window interactor. */
         vtkSmartPointer<vtkRenderWindowInteractor> interactor_;
+		
+		 /** \brief Removes a vtk actor from the screen.
+          * \param[in] actor a pointer to the vtk actor object
+          * \param[in] viewport the view port where the actor should be removed from (default: all)
+          */
+        bool
+		removeCustomActorFromRenderer(const vtkSmartPointer<vtkLODActor> &actor,
+                                 int viewport = 0)
+		{
+			removeActorFromRenderer (actor,viewport);
+						 
+		}
+        
+
+        /** \brief Removes a vtk actor from the screen.
+          * \param[in] actor a pointer to the vtk actor object
+          * \param[in] viewport the view port where the actor should be removed from (default: all)
+          */
+        bool
+		removeCustomActorFromRenderer (const vtkSmartPointer<vtkActor> &actor,
+                                 int viewport = 0)
+		{
+        removeActorFromRenderer (actor,viewport);
+		}
+		
+		 /** \brief Adds a vtk actor to screen.
+          * \param[in] actor a pointer to the vtk actor object
+          * \param[in] viewport the view port where the actor should be added to (default: all)
+          */
+        bool
+        removeCustomActorFromRenderer (const vtkSmartPointer<vtkProp> &actor,
+                                 int viewport = 0)
+		{
+			removeActorFromRenderer (actor,viewport); 
+		}
+		
+        /** \brief  Adds a vtk actor to screen.
+          * \param[in] actor a pointer to the vtk actor object
+          * \param[in] viewport port where the actor should be added to (default: 0/all)
+          *
+          * \note If viewport is set to 0, the actor will be added to all existing
+          * renders. To select a specific viewport use an integer between 1 and N.
+          */
+        void
```

Comment:
Try to maintain the formatting (despite visualization not being in formatting yet). Use spaces, and 2 space indents. Maintain proper indentation too.